app/src/main/java/io/homeassistant/companion/android/settings/shortcuts/ManageShortcutsSettingsFragment.kt

```diff
@@ -355,4 +401,28 @@ class ManageShortcutsSettingsFragment : PreferenceFragmentCompat() {
             }
         }
     }
+
+    override val iconDialogIconPack: IconPack
+        get() = iconPack
+
+    override fun onIconDialogIconsSelected(dialog: IconDialog, icons: List<com.maltaisn.icondialog.data.Icon>) {
+        Log.d(TAG, "Selected icon: ${icons.firstOrNull()}")
+        val selectedIcon = icons.firstOrNull()
+        if (selectedIcon != null) {
+            val iconDrawable = selectedIcon.drawable
+            if (iconDrawable != null) {
+                val icon = DrawableCompat.wrap(iconDrawable)
+                icon.setColorFilter(resources.getColor(R.color.colorAccent), PorterDuff.Mode.SRC_IN)
+                when (LAST_ICON_SELECT) {
```

Comment:
```suggestion
                when (dialog.tag) {
```
I think this is the syntax....